src/__tests__/SearchResults.test.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import SearchResults from "../components/SearchResults";
+
+describe("SearchResults", () => {
+  const validProps = {
+    results: ["image"],
```

Comment:
For this results array, why not use an array of strings instead of just one, as will often be the case with the data from the API?
```
results: [
  "test-url-1",
  "test-url-2",
  "test-url-3"
]
```
Not only can you then get a better snapshot of what the return will more likely be, but you can also assert the correct number of elements are being rendered based on the number of results you pass in.
You would have to update the test for 'renders an image' to use a different query that will look for multiple instances (`getAllByRole` should be fine for this)